tests/fields/test_decimal128_field.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from bson.decimal128 import Decimal128
+
+from mongoengine import *
+from tests.utils import MongoDBTestCase
+
+
+class TestDecimal128Field(MongoDBTestCase):
+    def test_decimal128_validation(self):
```

Comment:
separate the min/max validation tests from the rest, so that it's easier to understand what validation is being tested

Reply:
e31189061d89412242549324e2ec59c5845fe64b